roles/openshiftpostdeployment/tasks/main.yml

```diff
@@ -20,6 +20,28 @@
     - openshift
   when: installLogging == True
 
+- name: Label privileged namespaces
+  command: /usr/local/bin/oc label namespace {{ item }} state=privileged
+  with_items:
+    - default
+    - kube-service-catalog
+    - openshift-monitoring
+
+- name: Setup netpol files
+  template:
+    src: templates/netpol.j2
+    dest: ~/netpol.yml
+    force: yes
+    backup: yes
+
+- name: Register projects to create policy in
+  shell: /usr/local/bin/oc get projects | egrep -v '(default)|(openshift-monitoring)|(kube-service-catalog)|(NAME)' | awk '{print $1}'
+  register: netpol_projects
+
+- name: Create netpol objects in projects
+  command: /usr/local/bin/oc create -f ~/netpol.yml -n {{ item }}
```

Comment:
Instead of referencing a file could this be changed to use the ansible lookup module? https://docs.ansible.com/ansible/latest/plugins/lookup.html

Reply:
Lookup file not suitable for this - as it ruins the formatting